solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md

```diff
@@ -45,35 +45,32 @@ Notice that the answer must be a substring, "pwke" is a subsequence and not a su
 * `0 <= s.length <= 5 * 10^4`
 * `s` consists of English letters, digits, symbols and spaces.
 
-## Approach 1: Set
+## Approach 1: HashSet
 
-Create 2 pointer left and right place them at start of the string.
-Move right (right++) till distinct characters and store them in set.
-If repeated character occurs then move left (left++) until that repeated character is occured in left ans also remove all characters that occur before that character including character itself from set.
+Two pointer _i_ and _j_, initially at the start of the string.
+Move right (j++) till distinct characters and store them in set.
+If repeated character occurs then move left (i++) until that repeated character is occured in left ans also remove all characters that occur before that character including character itself from set.
+
+Time complexity: O(n), where n - # of characters in the string
+Space complexity: O(s), where s is the longest substring
```

Comment:
- Apply LaTex. 
- Use O instead of 0.
- Line break

Reply:
Modified the file and updated with the requested changes. Please verify once.